benchmarks/kotlin-ktor/README.md

```diff
@@ -0,0 +1,104 @@
+# Kotlin-Ktor benchmark
+
+The benchmark is designed to simulate a chat application where multiple clients send requests simultaneously.
+It uses the [Ktor](https://ktor.io/) framework.
+
+Namely, the benchmark runs multiple clients at the same time.
+Clients are set up with one or both of the following tasks:
+
+- Join a chat and send a randomized message to it.
+- Create a private chat with another user (if not already created) and send a randomized message to them.
+
+They repeat those task the required number of times.
```

Comment:
Could you clarify what is the required number of times? Is it `iterations_count`?

Typo: tasks